install_scripts/install_lago.sh

```diff
@@ -39,25 +39,26 @@ install_lago() {
 }
 
 add_lago_repo() {
+    if ! grep -q "Fedora" /etc/redhat-release; then
```

Comment:
Better swap the 'if' order and not use the '!' here. It will be more readble.

But the behaviour here is not reliable enough, what if `/etc/redhat-release` file is not there at all? We'll just assume its an 'el' system now... It better to use the more standard (since systemd) `/etc/system-release`, and still a separate check that the file is there at all is needed before trying to read and parse it.